test/KregielniaTests.cpp

```diff
@@ -9,3 +9,33 @@ TEST_F(KregielniaTests, assertThatTravisCiBuildStatusIsPass)
 {
     ASSERT_TRUE(true);
 }
+
+TEST_F(KregielniaTests, assertThatCanCreateTestFiles)
```

Comment:
You are not testing Kregielnia object here, but only filesystem library / TestFileSupport, so the test should be in another file dedicated to testing TestFileSupport

Reply:
Done. I created additional test case "InputFileSupportTests.cpp" for preparing dedicated test for that functionality.